src/bootstrap.ts

```diff
@@ -176,7 +177,7 @@ export const run = async (config: AppConfig) => {
 	}
 
 	// RxNorm Db
-	setupRxNormConnection(config.rxNormDbProperties());
```

Comment:
which in turn would make this `await` unnecessary... although it isn't immediately clear why that was added here at all, as the pool creation is not being awaited within the function 🤔 
perhaps I'm missing something here, but not all async functions need be awaited in their execution context.

Reply:
I've removed this await, this was there to accommodate previous version of `pingRxNorm`, also may have been early testing / debugging that wasn't cleaned up